src/parquet/util/rle-test.cc

```diff
@@ -318,14 +318,22 @@ TEST(BitRle, Flush) {
 // Test some random sequences.
 TEST(BitRle, Random) {
   int iters = 0;
-  while (iters < 1000) {
+
+  size_t niters = 500;
+  size_t ngroups = 1000;
+  size_t max_group_size = 16;
+  vector<int> values(ngroups + max_group_size);
+
+  while (iters < niters) {
```

Comment:
A style comment: `iters` is only used inside this loop, so it's cleaner to remove line 320 and use
`for (int iters=0; iters < niters; iters++) { srand(iters);`